docs/cluster-overview.md

```diff
@@ -52,8 +52,7 @@ The system currently supports three cluster managers:
 * [Apache Mesos](running-on-mesos.html) -- a general cluster manager that can also run Hadoop MapReduce
   and service applications.
 * [Hadoop YARN](running-on-yarn.html) -- the resource manager in Hadoop 2.
-* [Kubernetes](running-on-kubernetes.html) -- [Kubernetes](https://kubernetes.io/docs/concepts/overview/what-is-kubernetes/)
-is an open-source platform that provides container-centric infrastructure.
+* [Kubernetes](running-on-kubernetes.html) -- an open-source platform for container-centric infrastructure.
```

Comment:
Should we just use the intro on kubernetes.io? 
`open-source system for automating deployment, scaling, and management of containerized applications`.

Reply:
Done, good point, thanks!